templates/amm.cpp.in

```diff
@@ -492,7 +490,7 @@ Lepton, Fermion, Scalar
                const double part1 = coeffA.real() * Sqr(scalarMass) / (Sqr(scalarMass) - Sqr(zMass)) * (BarrZeeLoopFS(massRatioSquared) - BarrZeeLoopFS(Sqr(fermionMass / zMass)));
                const double part2 = coeffB.real() * Sqr(scalarMass) / (Sqr(scalarMass) - Sqr(zMass)) * (BarrZeeLoopFPS(massRatioSquared) - BarrZeeLoopFPS(Sqr(fermionMass / zMass)));
 
-               const double preFactor = - unit_charge(context)/(64*Power4(Pi)) * fermionChargeCount * gvm * gvf
+               const double preFactor = -1.0/(64*Power4(Pi)) * fermionChargeCount * gvm * gvf
```

Comment:
I'd propose to write
~~~.cpp
-4*twoLoop
~~~
for clarity, where `twoLoop` is the constant from `wrappers.hpp`.

Reply:
Done.